src/components/pages/home/sanity-sections.tsx

```diff
@@ -0,0 +1,188 @@
+                  resource={resource}
+                  location={location}
+                  completedCoursesIds={completedCoursesIds}
+                />
+              )
+            case 7:
+              return i === 0 ? (
+                <HorizontalResourceCard
+                  className="col-span-2"
+                  key={resource.id}
+                  resource={resource}
+                  location={location}
+                  completedCoursesIds={completedCoursesIds}
+                />
+              ) : (
+                <VerticalResourceCard
+                  key={resource.id}
+                  resource={resource}
+                  location={location}
+                  completedCoursesIds={completedCoursesIds}
+                />
+              )
+            default:
+              return (
+                <VerticalResourceCard
+                  key={resource.id}
+                  resource={resource}
+                  location={location}
+                  completedCoursesIds={completedCoursesIds}
+                />
+              )
+          }
+        })}
+      </Grid>
+      {section.path && (
+        <div className="flex justify-end mt-3">
+          <Link href={section.path} passHref>
+            <a
+              onClick={() => {
+                analytics.events.activityInternalLinkClick(
+                  'curated topic page',
+                  'home page-curated section',
+                  section.title,
+                  section.path,
+                )
+              }}
+              className="flex items-center px-4 py-3 text-sm transition-all duration-200 ease-in-out bg-transparent border-b-2 border-gray-200 dark:border-gray-800 border-opacity-70 dark:hover:bg-gray-800 dark:hover:bg-opacity-50 opacity-80 hover:opacity-100 group"
+            >
+              Browse all{' '}
+              <span
+                className="pl-1 transition-all duration-200 ease-in-out group-hover:translate-x-1"
+                aria-hidden
+              >
+                →
+              </span>
+            </a>
+          </Link>
+        </div>
+      )}
+    </section>
+  )
+}
+
+const SanitySections = ({
+  sections,
+  location,
+  completedCoursesIds,
+}: {
+  sections: SanitySectionType[]
+  location?: string
+  completedCoursesIds?: number[]
+}) => {
+  return (
+    <>
+      {sections
+        .filter((s: any) => s.slug !== 'jumbotron')
+        .map((section: any, i: number) => {
+          switch (true) {
+            case section.slug === 'topics':
+              return <Topics data={section} key={i} />
+            case section.displayComponent === 'eggheadForTeamsCta':
+              return <EggheadForTeamsCta />
+          }
+
+          return (
+            <DynamicGridComponent
+              section={section}
+              key={i}
+              location={location}
+              completedCoursesIds={completedCoursesIds}
+            />
+          )
+        })}
```

Comment:
switch on topics (that was behavior we already had in the home page so left it), and `section.displayComponent`. 

We could add more `displayComponent` types in sanity if we wanted to customize sections.

Default is the `DynamicGridComponent` we are used to.